must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -444,12 +445,17 @@ CMDS
         crash_collection &
         pids_log+=($!)
         journal_collection &
-        pids_log+=($!)
+        pids_journal+=($!)
         if [ -n "${pids_log[*]}" ]; then
             # wait for all pids
             echo "waiting for ${pids_log[*]} to terminate" | tee -a  "${BASE_COLLECTION_PATH}"/gather-debug.log
             wait "${pids_log[@]}"
         fi
+        if [ -n "${pids_journal[*]}" ]; then
+            # wait for all pids
+            echo "waiting for ${pids_journal[*]} to terminate" | tee -a  "${BASE_COLLECTION_PATH}"/gather-debug.log
+            wait "${pids_journal[@]}"
+        fi
         delete_journal_log
```

Comment:
I don't think this is the right place to call `delete_journal_log`.  Here it's called for *each* node -- when it already iterates over all nodes internally.  See my previous comment:
>  Perhaps you meant to move it outside the loop (since it has a loop of its own) or alternatively make `delete_journal_log` operate on a single node?

Reply:
sorry, missed it. Thanks.

Reply:
I'd suggest renaming this function to `delete_journal_logs` so that there is at least some hint in the name.  Or, better yet, just open-code it similar to journal log generation which is also open-coded: https://github.com/red-hat-storage/ocs-operator/blob/fb00fe37905a8a66e52b79cf97e825734aa9cedb/must-gather/collection-scripts/gather_ceph_resources#L419-L428

Reply:
This is not deleting the logs, it's about generating the logs, open code to this seems fine to me.

Reply:
The print msg describes it all..Let me know what you think @idryomov

Reply:
> The print msg describes it all..Let me know what you think @idryomov

What print msg?

Reply:
`printf "Generating journal logs from node %s \n"  "${node}" | tee -a  "${BASE_COLLECTION_PATH}"/gather-debug.log `

Reply:
line 420 in above code

Reply:
I understand the different between generation and deletion ;)

What I was pointing out is that journal log generation is currently open-coded, while journal log deletion is wrapped into a function with a rather confusing name which is probably what caused you to place it into the loop body by mistake.  I'm suggesting to open-code journal log deletion as well -- thus making that mistake very hard to repeat.